os/kernel/log_dump/log_dump.c

```diff
@@ -33,6 +33,10 @@
 #include <queue.h>
 
 #include <tinyara/log_dump/log_dump.h>
+#if defined(CONFIG_LIBMINIZ)
+#include "miniz/miniz.h"
+#endif
+#include <tinyara/lzma/LzmaLib.h>
```

Comment:
1. Need #if conditional for lzma
2. Could you check why including method is different with minz and lzma? I think we should make them using the same way.

Reply:
yes will raise new PR for the  miniz cleanup